R/cohensFleissKappa.R

```diff
@@ -192,3 +193,59 @@ cohensFleissKappa <- function(jaspResults, dataset, options) {
   }
   return(jaspTable)
 }
+
+.computeKrippendorffsAlphaTable <- function(dataset, options, ready) {
+  # Create the JASP Table
+  jaspTable <- createJaspTable(title = gettextf("Krippendorff's alpha"))
+  jaspTable$addColumnInfo(name = "method", title = gettext("Method"), type = "string")
+  jaspTable$addColumnInfo(name = "kAlpha", title = gettext("Krippendorff's alpha"), type = "number")
+  jaspTable$position <- 2
+  
+  #dependencies
+  jaspTable$dependOn(
+    options = c(
+      "variables",
+      "krippendorffsAlpha",
+      "kappaIntervalOn",
+      "kappaConfidenceIntervalValue"
+    )
+  )
+  
+  formattedCIPercent <- format(
+    100 * options[["kappaConfidenceIntervalValue"]],
+    digits = 3,
+    drop0trailing = TRUE
+  )
+  
+  if (ready) {
+    #calculate Krippendorff's alpha
+    kAlphaData <- t(as.matrix(dataset))
```

Comment:
this should definitely be explained in the help file, since apparently the "usual" data structure for the alpha has raters in the rows... unlike what we do here and for the other coefficients.